src/main/resources/static/css/style2.css

```
@@ -226,3 +226,65 @@ body {
+}
+
+.user-tooltip-role {
+    font-size: 13px;
+    color: #6366f1;
+    margin-bottom: 4px;
+}
+
+.user-tooltip-email {
+    font-size: 12px;
+    color: #64748b;
+    word-break: break-word;
```

Stylelint (17.7.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)